docs/reference/groups.yml

```diff
@@ -170,8 +170,8 @@
   category: symbols
   path: ["emoji"]
   details: |
-    Named emoji.
+    Named emojis.
```

Comment:
I could have sworn than `emoji` is its own plural, but [this Reddit comment](https://www.reddit.com/r/asklinguistics/comments/6li4lz/comment/dju73z0/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button) convinced me that it's not the case.